### PR TITLE
Table checkbox

### PR DESCRIPTION
**Pull request by NikitaPoly**

### CODE REVIEW

File: frontend/pulse3d/src/pages/uploads.js [hunk 1]
```diff
@@ -14,25 +14,30 @@ const uploadTableColumns = [
...
   },
   {
     name: "Created Date",
     admin: false,
+    center: false,
     selector: (row) => row.createdAt,
   },
   {
     name: "Last Analyzed",
+    center: false,
```
Comment (tannermpeterson) [inline frontend/pulse3d/src/pages/uploads.js:40]: If these should all be `false` now, you can just change the value on line 48

File: frontend/pulse3d/src/pages/users-info.js [hunk 2]
```diff
@@ -9,22 +9,38 @@ import CircularSpinner from "@/components/basicWidgets/CircularSpinner";
...
     name: "Date Created",
+
+    center: false,
+    sortable: true,
+
     selector: (row) => formatDateTime(row.created_at),
   },
   {
     name: "Last Loggedin",
+    center: false,
```
Comment (tannermpeterson) [inline frontend/pulse3d/src/pages/users-info.js:42]: these values are defined on line 49 now

File: frontend/pulse3d/src/pages/uploads.js [hunk 3]
```diff
@@ -273,36 +278,31 @@ export default function Uploads() {
   useEffect(() => {
     getAllJobs();
     // start 10 second interval
-    const uploadsInterval = setInterval(() => getAllJobs(), [1e4]);
```
Comment (luciipak) [inline frontend/pulse3d/src/pages/uploads.js:276]: How come we want to remove this?

File: frontend/pulse3d/src/pages/uploads.js [hunk 4]
```diff
@@ -273,36 +278,31 @@ export default function Uploads() {
   useEffect(() => {
     getAllJobs();
     // start 10 second interval
```
Comment (luciipak) [inline frontend/pulse3d/src/pages/uploads.js:280]: can you remove this comment

File: frontend/pulse3d/src/pages/uploads.js (hunk 3 above)
Comment (tannermpeterson) [inline frontend/pulse3d/src/pages/uploads.js:276]: If this is getting removed in favor of another method of polling the job status, then I don't think it is working currently. If there is new code to do this, it needs to be fixed, or this should just be added back

**Review by tannermpeterson — CHANGES_REQUESTED** [review]

**Pull request closed without merging**